frontend/src/api/k8s/routes.ts

```diff
@@ -20,5 +20,8 @@ export const getServiceMeshGwHost = async (namespace: string): Promise<string |
     name: namespace,
   };
   const project = await k8sGetResource<NamespaceKind>({ model: NamespaceModel, queryOptions });
-  return project?.metadata?.annotations?.['opendatahub.io/service-mesh-gw-host'] || null;
+  return (
+    project?.metadata?.annotations?.['service-mesh.opendatahub.io/public-gateway-host-external'] ||
```

Comment:
As we have `service-mesh.opendatahub.io/public-gateway-host-external` already in two places can we add it somewhere as a `const`?

Reply:
I don't think so - the two references are found in the `frontend/` and `backend/` sections of the codebase respectively - and I don't think that the two share consts (from what I can see).